### ISSUE TITLE
`default` image doesn't work - returns 404 regardless

### ISSUE THREAD

**Issue by noway** [andrieslouw/imagesweserv#210]
Was working fine before. Seems to be a regression. Example:

```
https://t0.nl/?url=https%3A%2F%2Ft0.nl%2F%3Furl%3Dhttps%253A%252F%252Fplatform-lookaside.fbsbx.com%252Fplatform%252Fprofilepic%252F%253Fasid%253D1756318947821786%2526width%253D640%2526ext%253D1575594837%2526hash%253DAeSElpWFBDGplqOO%26default%3Dhttps%253A%252F%252Ft0.nl%252F%253Furl%253Dhttps%25253A%25252F%25252Fs3-ap-southeast-2.amazonaws.com%25252Fturnout-misc%25252Ffallback_avatar_400x400.png%2526w%253D64%2526h%253D72%2526t%253Dcontain%2526mask%253Dcircle%2526mbg%253DFFFFFF%26w%3D64%26h%3D72%26t%3Dcover%26mask%3Dcircle%26mbg%3DFFFFFF&w=72&h=72&t=contain&bg=FFFFFF&mask=circle&id=friend-1d5fc35e-de0c-4b0a-8d93-a8ed07a50f4e
```

**Comment by andrieslouw** [comment]
Shouldn't it be:
https://t0.nl/?url=https%3A%2F%2Fplatform-lookaside.fbsbx.com%2Fplatform%2Fprofilepic%2F%3Fasid%3D1756318947821786%26width%3D640%26ext%3D1575594837%26hash%3DAeSElpWFBDGplqOO&default=https%3A%2F%2Ft0.nl%2F%3Furl%3Dhttps%253A%252F%252Fs3-ap-southeast-2.amazonaws.com%252Fturnout-misc%252Ffallback_avatar_400x400.png%26w%3D64%26h%3D72%26t%3Dcontain%26mask%3Dcircle%26mbg%3DFFFFFF&w=64&h=72&t=cover&mask=circle&mbg=FFFFFF&w=72&h=72&t=contain&bg=FFFFFF&mask=circle&id=friend-1d5fc35e-de0c-4b0a-8d93-a8ed07a50f4e

**Comment by andrieslouw** [comment]
Also, please don't use t0.nl in production, it is our testing environment, and may be highly unstable. Use images.weserv.nl instead, like this:
https://images.weserv.nl/?url=https%3A%2F%2Fplatform-lookaside.fbsbx.com%2Fplatform%2Fprofilepic%2F%3Fasid%3D1756318947821786%26width%3D640%26ext%3D1575594837%26hash%3DAeSElpWFBDGplqOO&default=https%3A%2F%2Fimages.weserv.nl%2F%3Furl%3Dhttps%253A%252F%252Fs3-ap-southeast-2.amazonaws.com%252Fturnout-misc%252Ffallback_avatar_400x400.png%26w%3D64%26h%3D72%26t%3Dcontain%26mask%3Dcircle%26mbg%3DFFFFFF&w=64&h=72&t=cover&mask=circle&mbg=FFFFFF&w=72&h=72&t=contain&bg=FFFFFF&mask=circle&id=friend-1d5fc35e-de0c-4b0a-8d93-a8ed07a50f4e

**Comment by noway** [comment]
These urls are incomplete and don't show the proper image. I do a multi pass of t0.nl urls to properly crop the image. Here's I got it working again:
```
https://t0.nl/?url=ssl%3At0.nl%2F%3Furl%3Dssl%253Aplatform-lookaside.fbsbx.com%252Fplatform%252Fprofilepic%252F%253Fasid%253D1756318947821786%2526width%253D640%2526ext%253D1575594837%2526hash%253DAeSElpWFBDGplqOO%26default%3Dssl%253At0.nl%252F%253Furl%253Dssl%25253As3-ap-southeast-2.amazonaws.com%25252Fturnout-misc%25252Ffallback_avatar_400x400.png%2526w%253D64%2526h%253D72%2526t%253Dcontain%2526mask%253Dcircle%2526mbg%253DFFFFFF%26w%3D64%26h%3D72%26t%3Dcover%26mask%3Dcircle%26mbg%3DFFFFFF&w=72&h=72&t=contain&bg=FFFFFF&mask=circle&id=friend-1d5fc35e-de0c-4b0a-8d93-a8ed07a50f4e
```
The seems to be an issue with `https://` in particular. replacing `https://` to `ssl:` seems to solve it. Perhaps you started rewriting somehow automatically https:// to ssl:? I honestly feel myself like character of this comics https://xkcd.com/1172/

>please don't use t0.nl in production, it is our testing environment, and may be highly unstable. Use images.weserv.nl instead

Ok, no worries, changed my code to images.weserv.nl.

(You can close this issue if you think it's not worth pursuing changing the behaviour back)

**Comment by kleisauke** [comment]
It seems that the (Amazon S3) fallback URI timed-out on 2020/01/17, this error was cached for a long period because I forgot to roll out the fix mentioned in #207 on t0.nl.

I've just purged all the cache and changed the Edge-Cache TTL setting on this testing domain, so the above-mentioned URI's should work again. It's indeed not recommended to use t0.nl in production.

**Comment by noway** [comment]
Omg, it does work now! Seems like that was the issue indeed!
Thank you!
